src/ai/backend/agent/agent.py

```diff
@@ -268,8 +268,9 @@ async def generate_accelerator_mounts(
     ) -> List[MountInfo]:
         raise NotImplementedError
 
+    @staticmethod
     @abstractmethod
-    def resolve_krunner_filepath(self, filename) -> Path:
+    def resolve_krunner_filepath(filename) -> Path:
```

Comment:
If there is no good reason, I think it's better to keep this an instance method, to allow future possibilites of accessing self's states in other subclass implementations.